### ISSUE TITLE
Properly stringify error

### ISSUE THREAD

**Issue by MartinMuzatko** [strapi/strapi-sdk-javascript#91]
Related to #47 

**Informations**
- **Node.js version**: 12.14.0
- **npm version**: 6.13.4
- **Strapi version**: beta-19
- **Strapi-sdk-javascript version**: 0.3.3
- **Browser or Node.js**: NodeJS

**What is the current behavior?**

Right now, I can only retrieve `[Object, object]` as error message, because `Error`s in JavaScript are stringified before stored in the Error object.
https://github.com/strapi/strapi-sdk-javascript/blob/master/src/lib/sdk.ts#L99

**Steps to reproduce the problem**

e.g. register without passing email, anything that is producing a non 2xx/3xx status code

**What is the expected behavior?**

I should be able to retrieve the complete response and response body.
Alternatively every known error should be its own Error class, since multiple things can go wrong, this is not always feasible (e.g. username and email missing errors)

**Suggested solutions**

I propose these solutions. Pick one or multiple.
* Add `.toString` method to the received response to e.g. instead do JSON.stringify
* Attach custom data to the error. e.g. `error.response`

**Confirm**
- [X] I'm sure that this issue hasn't already been referenced

**Comment by Marian0** [comment]
I'm having the same problem. I cannot get the error from the API using this Sdk. Do you have any suggestion to handle this problem ?

**Comment by MartinMuzatko** [comment]
@Marian0 My suggestions are listed above in "Suggested solutions" part.

You could either fork strapi sdk and fix it for you, or you need to use axios instead. I just saw that the sdk is deprecated and unmaintained anyway :O

**Comment by Marian0** [comment]
@MartinMuzatko you are right. Seems like the best choice is to create our own strapi service using axios. If this week have some time I can create it and let you take a look. :D

**Comment by bitbytebit1** [comment]
@Marian0 also looking for this functionality!!!
